TLS.md

```diff
@@ -9,7 +9,7 @@ Getting Started
 To build with TLS support you'll need OpenSSL development libraries (e.g.
 libssl-dev on Debian/Ubuntu).
 
-To build TLS support as Redis built-in:
+To build TLS support as Redis Community Edition built-in:
```

Comment:
this is probably excessive too.. but what you may wanna change is the text in `asciilogo.h`

Reply:
Changed asciilogo.h as well.
See my not above